### PR TITLE
Fix naming of set_dps in pytuya

### PR DESCRIPTION
**Pull request by postlund**
Renames `set_dps` to `set_dp` and introduces `set_dps` which can set multiple datapoints at once (needed for RGB support).

### CODE REVIEW

**Review by rospogrigio — COMMENTED** [review]
Why not change everything, at this point? I mean changing also dps_index to dp_index and _dps_id to _dp_id ...

**Comment by postlund** [comment]
You are right, might as well do that.

File: custom_components/localtuya/pytuya/__init__.py [hunk 1]
```diff
@@ -457,12 +461,12 @@ async def detect_available_dps(self):
         self.log.debug("detected dps: %s", detected_dps)
         return detected_dps
 
-    def add_dps_to_request(self, dps_index):
+    def add_dps_to_request(self, dp_index):
```
Comment (rospogrigio) [inline custom_components/localtuya/pytuya/__init__.py:464]: Here I would leave dps_index (or dp_indices), since this can actually be a list of DPs...

**Comment by postlund** [comment]
Now I have renamed more stuff.

**Review by rospogrigio — APPROVED** [review]

**Comment by rospogrigio** [comment]
Merged #83.

**Pull request merged**